backend/api/Controllers/InspectionAreaController.cs

```diff
@@ -219,7 +257,7 @@ await inspectionAreaService.ReadByInstallationAndPlantAndName(
             catch (Exception e)
```

Comment:
Suggest catching InvalidPolygonException before generic catch to improve error message